pkg/api/series.go

```diff
@@ -71,12 +71,12 @@ func series(queryable *query.Queryable) http.HandlerFunc {
 		var sets []storage.SeriesSet
 		var warnings storage.Warnings
 		for _, mset := range matcherSets {
-			s, _, wrn, err := q.Select(false, nil, nil, mset...)
-			warnings = append(warnings, wrn...)
-			if err != nil {
-				respondError(w, http.StatusUnprocessableEntity, err, "execution")
-				return
-			}
+			s, _ := q.Select(false, nil, nil, mset...)
+			//warnings = append(warnings, wrn...)
```

Comment:
can't we grab these warning from s.Warnings() now?

Reply:
Ahh, I missed those debug comments. Anyways, the warnings will now be handled in the SeriesSet. I kept Warnings as nil in pgxSeriesSet since in upstream, the `blockBaseSeriesSet` has the same behaviour.

But still, being on the safe side, we can make a way to pass warnings through `(*pgxSeriesSet).Warnings()`.

Reply:
Yes let's do that. (pass it up)

Reply:
(even if we do only ever return `nil` in our current `SeriesSet` we should still retain the logic to forward the warnings here for ease of future changes)

Reply:
Agreed.